app/src/services/activity.ts

```diff
@@ -62,6 +63,31 @@ const service = {
     });
 
     return response ? activity.fromPrismaModel(response) : null;
+  },
+
+  /**
+   * @function getActivities
+   * Get all activities (not deleted)
+   * If given an initiative code, only return activities for that initiative's projects
+   * @param {string} [initiative] Optional initiative code
+   * @returns {Promise<Activity[]>} The result of running the findMany operation
+   */
+  getActivities: async (initiative?: Initiative) => {
```

Comment:
We've historically been inconsistent, but would be good to make sure we have explicit type returns in typescript format over needing to depend on JSDoc for it:
```ts
  getActivities: async (initiative?: Initiative): Primise<Activity[]> => {
```